.github/workflows/release_actions.yml

```
@@ -0,0 +1,53 @@
+          context: .
+          push: true
+          tags: 
+            $IMAGE_NAME:latest
+            $IMAGE_NAME:${{ steps.get_version.outputs.VERSION }}
+
+  push-latest-tag:
+    runs-on: ubuntu-latest
+    needs: publish-docker-image
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)